.travis.yml

```diff
@@ -192,6 +188,10 @@ jobs:
       - gpg --list-keys
       - bin/release_android "$TRAVIS_TAG"
       name: "Publish Android SDK to the Sonatype and Maven Central"
+    - script:
+      - bin/travis_scripts/trigger_desktop_app_build
+      name: "Trigger Desktop APP build"
+      if: tag = env(BUILD_DEV_RELEASE)
```

Comment:
why we build just for dev release?

Reply:
we are triggering a build for the master branch only:
https://github.com/mysteriumnetwork/node/blob/master/bin/travis_scripts/trigger_desktop_app_build#L5

I think there is no point to trigger a desktop app master build for other releases.

Reply:
We probably should have automated trigger not for master only in the future, but thats for @ignasbernotas to decide. Lets leave it like this for now.